src/SFA.DAS.EmployerApprenticeshipsService.Application.UnitTests/Commands/DeleteTransferConnectionInvitation/WhenIDeleteATransferConnectionInvitation.cs

```diff
@@ -154,11 +155,39 @@ public void Arrange()
         }
 
         [Test]
-        public void ThenShouldThrowExceptionIfDeleterIsNotTheSender()
+        public async Task ThenShouldSetSenderDeletedWhenAccountIsSender()
```

Comment:
The unit tests can now be moved to https://github.com/SkillsFundingAgency/das-employerapprenticeshipsservice/blob/master/src/SFA.DAS.EmployerApprenticeshipsService.Domain.UnitTests/Models/TransferConnectionInvitationTests.cs 😃